packages/auth/blueprints/spree-ember-auth-components/files/app/components/spree-auth-orders.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+import layout from '../templates/components/spree-auth-orders';
+/**
+ A order list for the Spree Account Route.
+
+ **To Override:** You'll need to run the components generator:
+
+ ```bash
+ ember g spree-ember-auth-components
+ ```
+
+ This will install a layout file at: `app/templates/components/spree-auth`, and
+ a component file at `app/components/spree-auth`, ready to be extended.
+
+ @class SpreeAccountOrders
+ @namespace Component
+ @extends Ember.Component
+ */
+export default Ember.Component.extend({
+  layout: layout
+});
```

Comment:
If you're worried about having a big chunk of markup on the /account page, it would be fine to simplify the table.  It's really up to the developer as to how much data they want to display there.  We're mainly just concerned with exposing the data, and the `link-to` orders show page